src/k8s-engine/lima.ts

```diff
@@ -23,9 +23,11 @@ import ProgressTracker from './progressTracker';
 import * as K8s from './k8s';
 import { ContainerEngine, Settings } from '@/config/settings';
 import * as childProcess from '@/utils/childProcess';
+import clone from '@/utils/clone';
```

Comment:
I see 5 places in the code where we do `JSON.parse(JSON.stringify(X))` -- they should be replaced with `clone` as well.

Reply:
Fixed in 37ddec60ec2777ac05267cf44bdbaeb2ef2e8c98